common/src/main/java/pl/edu/agh/messages/client_communication/ClientMessage.java

```diff
@@ -0,0 +1,9 @@
+package pl.edu.agh.messages.client_communication;
+
+import pl.edu.agh.messages.RaftMessage;
+
+/**
+ * Created by Andrzej on 2016-11-03.
```

Comment:
Why should I remove it? RaftMessage is in different package and I extend it here